api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -84,7 +84,7 @@
   // Importing the real one requires importing a large subtree of dependencies
   @Bean
   public WorkspaceService workspaceService() {
-    return new WorkspaceServiceImpl(null, null, null, null, null, null, null, null, null);
+    return new WorkspaceServiceImpl(null, null, null, null, null, null, null, null, null, null);
```

Comment:
I was going to post some reaction gifs to this constructor invocation, but I think there's an easier way to solve this issue.

I'd go ahead and make a `FakeWorkspaceServiceImpl` class, use IntelliJ's default overload implementations that just return null, give it a zero-arg constructor, and tag it with `@Service` and give it a bean name. Then just inject it like a real service.

You'll likely need to mark the real one as primary so that the fake doesn't get pulled into real code.

Even simpler would be to mock it with Mockito, but I think using it in production code is controversial if not frowned upon.

It's still a smell to me that you need to pull in this bean but don't need a working version, but I haven't looked around very much to understand the main issue you're working around.

Reply:
Right, so you need something depends on WorkspaceService, but aren't actually calling any method on it that needs WorkspaceService. That's a symptom of the services getting too large most likely. Not something to chase here if you're trying to keep this small though.